astropy/visualization/wcsaxes/core.py

```diff
@@ -326,6 +326,63 @@ def plot_coord(self, *args, **kwargs):
 
         return super().plot(*args, **kwargs)
 
+    def scatter_coord(self, *args, **kwargs):
+        """
+        Scatter `~astropy.coordinates.SkyCoord` or
+        `~astropy.coordinates.BaseCoordinateFrame` objects onto the axes.
+
+        The first argument to
+        :meth:`~astropy.visualization.wcsaxes.WCSAxes.plot_coord` should be a
+        coordinate, which will then be converted to the first two parameters to
+        `matplotlib.axes.Axes.scatter`. All other arguments are the same as
+        `matplotlib.axes.Axes.scatter`. If not specified a ``transform``
+        keyword argument will be created based on the coordinate.
+
+        Parameters
+        ----------
+        coordinate : `~astropy.coordinates.SkyCoord` or `~astropy.coordinates.BaseCoordinateFrame`
+            The coordinate object to scatter on the axes. This is converted to
+            the first two arguments to `matplotlib.axes.Axes.scatter`.
+
+        See Also
+        --------
+
+        matplotlib.axes.Axes.scatter : This method is called from this function with all arguments passed to it.
+
+        """
+
+        if isinstance(args[0], (SkyCoord, BaseCoordinateFrame)):
```

Comment:
Rather than replicating this whole block from `plot_coord` can we make it a private helper method called from both places?

Reply:
Good idea.  I just committed an update that puts the duplicated code in a private method ``_transform_plot_args``.